runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -675,7 +675,7 @@ public void testBasicHarness() throws Exception {
           makeExpectedOutput(i, TimeUnit.MILLISECONDS.toMicros(i)).build(), result.get((long) i));
     }
 
-    verify(hotKeyLogger, atLeastOnce()).logHotKeyDetection(anyString(), any());
+    verify(hotKeyLogger, atLeastOnce()).logHotKeyDetection(nullable(String.class), any());
```

Comment:
Just can you explain the addition of the `nullable`? Was it always nullable and now it is more strict? Or is this not needed boilerplate from the migration?

Reply:
`anyString()` was always nullable. Apparently the test in question was passing `null` here instead of a `String`.

Reply:
SGTM